types/react-currency-formatter/index.d.ts

```diff
@@ -14,6 +14,7 @@ interface CurrencyFormatterProps {
 	pattern?: string;
 	decimal?: string;
 	group?: string;
+    symbol?: string;
```

Comment:
The rest of this file appears to be using tabs.

Reply:
Updated whitespace to follow what came before.